src/material/radio/radio.ts

```diff
@@ -462,13 +481,19 @@ export class MatRadioButton extends _MatRadioButtonMixinBase
               private _changeDetector: ChangeDetectorRef,
               private _focusMonitor: FocusMonitor,
               private _radioDispatcher: UniqueSelectionDispatcher,
+              @Optional() @Inject(MAT_RADIO_DEFAULT_OPTIONS)
+              private _providerOverride?: MatRadioDefaultOptions,
```

Comment:
New optional constructor arguments need to go at the end in case someone is already calling `super` with the existing arguments

Reply:
Done